controller.go

```diff
@@ -258,7 +258,7 @@ func (c *Controller) Abort(code string) {
 
 // CustomAbort stops controller handler and show the error data, it's similar Aborts, but support status code and body.
 func (c *Controller) CustomAbort(status int, body string) {
-	c.Ctx.ResponseWriter.WriteHeader(status)
+	c.Ctx.Output.Status = status
```

Comment:
@xionglun thanks for your good question. This is a known bug and has bee fixed #1805 ,and there is farther talk #1822